bin/generate

```diff
@@ -6,7 +6,7 @@ set -ea
 
 CMD_USAGE="Generate network configs
 
-Usage: generate <network_name> <masternodes_count>"
+Usage: generate <network_name> <masternodes_count> <seeds_count>"
```

Comment:
What if we change seed count from argument to option because in 99% you don't want to change and default value (2) is good enough.

Reply:
Does this look right? I tried to put the options before the arguments but I can't understand the bash syntax, even like this I'm 100% sure if `for i in "${@:3}"` is correct - I can't break it, but I also can't figure out what it actually does.